src/cryptography/hazmat/backends/openssl/decode_asn1.py

```diff
@@ -857,6 +857,12 @@ def _decode_nonce(backend, nonce):
     OCSPExtensionOID.NONCE: _decode_nonce,
 }
 
+_OCSP_SINGLERESP_EXTENSION_HANDLERS_NO_SCT = _REVOKED_EXTENSION_HANDLERS.copy()
+_OCSP_SINGLERESP_EXTENSION_HANDLERS = _REVOKED_EXTENSION_HANDLERS.copy()
+_OCSP_SINGLERESP_EXTENSION_HANDLERS[
+    ExtensionOID.PRECERT_SIGNED_CERTIFICATE_TIMESTAMPS
+] = _decode_precert_signed_certificate_timestamps
```

Comment:
I don't think this is correct -- this shouldn't be a PrecertedSignedCerificateTimestamps, that's for placing in an X.509 certificate. The OCSP extension is different.